snakes_cafe/snakes_cafe.py

```diff
@@ -0,0 +1,55 @@
+print("""
+$ python snakes_cafe.py
+**************************************
+**    Welcome to the Snakes Cafe!   **
+**    Please see our menu below.    **
+**                                  **
+** To quit at any time, type "quit" **
+**************************************
+""")
+appetizers = ['Wings', 'Cookies', 'Spring Rolls']
+entrees = ['Salmon', 'Steak', 'Meat Tornado', 'A Literal Garden']
+deserts = ['Ice Cream', 'Cake', 'Pie']
+drinks = ['Coffee', 'Tea', 'Unicorn Tears']
+
+print("""Appetizers 
+----------""")
+for appetizer in appetizers:
+    print(appetizer)
+
+print("""Entrees 
+-------""")
+for entree in entrees:
+    print(entree)
+
+print("""Deserts 
+-------""")
+for desert in deserts:
+    print(desert)
+
+print("""Drinks
+------""")
+for drink in drinks:
+    print(drink)
+
+print("""
+***********************************
+** What would you like to order? **
+***********************************
+""")
+menu = {}
+
+while True:
+	item = input("> ")
+	if item == 'quit':
+		break
+	elif item not in menu:
+		menu[item] = 0
+	menu[item] += 1
+	print(f"{menu[item]}orders of {item} have been added to your meal")

```

Comment:
You've got the basic concept down here, but remember that per the lab example we want different responses based on whether we have more than one of an item ordered:

```
#Example output

***********************************
** What would you like to order? **
***********************************
> Wings

** 1 order of Wings have been added to your meal **

> Wings

** 2 orders of Wings have been added to your meal **
```

Consider introducing some conditional logic that checks the count of an item in your menu dict and decides the response accordingly.